stories/guides/component-dependencies.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import LinkTo from '@storybook/addon-links/react';
```

Comment:
Ill swing around after and put some of the context stuff in here as they can be viewed as dependencies